.travis.yml

```diff
@@ -16,13 +15,13 @@ notifications:
   email: false
 
 after_success:
-  - julia -e 'include(Pkg.dir("DocStringExtensions", "test", "coverage.jl"))'
+  - julia -e 'include("test/coverage.jl")'
```

Comment:
Just `julia test/coverage.jl` maybe?

Reply:
Also, the coverage script needs to be updated as well (it doesn't import Pkg at the moment).